content/en/platform/corda/4.10/community/release-checksum-os.md

```diff
@@ -15,3 +15,657 @@ Before you install the downloaded files, verify the integrity of each file to en
 
 ## Downloads for release 4.10
 
+Downloads for release-os-4.10
```

Comment:
Is this a duplication of the heading above?

Reply:
Just pushed an update there.